src/Providers/StoreFileProvider.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Providers;
+
+use Blumilk\Meetup\Core\Contracts\StoreFile;
+use Blumilk\Meetup\Core\Services\StoreFileService as Service;
+use Illuminate\Support\ServiceProvider;
+
+class StoreFileProvider extends ServiceProvider
```

Comment:
I think that it is needed, because we have file support in several places and there will be more of them. I try binding a concreate instance for a default laravel behavior and not working